Kitodo/src/main/java/org/kitodo/enums/FilterString.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.enums;
+
+public enum FilterString {
+
+    PROCESSPROPERTY("processproperty:", "prozesseigenschaft:"), TASK("step:", "schritt:"), TASKINWORK("stepinwork:",
```

Comment:
Formatting of this enums looks strange and it is complicate to read or to adjust. Maybe formatting rules must be adjusted or?

Reply:
Resolved in #956